src/interface/rest/middlewares/validateRecaptcha.js

```diff
@@ -23,8 +23,8 @@ async function validateRecaptcha(req, res, next) {
     next();
   } else {
     return ErrorHandler.throwError({
-      code: 401,
-      message: `RecaptchValidation Failed`,
+      code: 403,
```

Comment:
code should bee 429 here.
You can check HTTP Codes to understand which one is used for what. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status